### PR TITLE
Add sepolia config

### PR DESCRIPTION
**Pull request by Wizdave97**
This PR replaces the goerli sync committee configs with those for sepolia.
It also updates all ismp related contract addresses to the equivalent sepolia deployments.

### CODE REVIEW

File: client/src/main.rs [hunk 1]
```diff
@@ -106,7 +106,7 @@ async fn main() -> Result<(), anyhow::Error> {
                         token_contract: MULTICHAIN_TOKEN,
                         timeout: 3 * 60 * 60,
                     })
-                    .gas(100_000)
+                    .gas(1_000_000)
```
Comment (seunlanlege) [inline client/src/main.rs:109]: Why 1m gas?

Comment (Wizdave97) [reply client/src/main.rs:109]: I got an out of gas error when I ran it with the 100k gas, I'll revert it back and try again, just want to test the deployments first

Comment (Wizdave97) [reply client/src/main.rs:109]: This was the tx https://sepolia.etherscan.io/tx/0x4c4bf173237d31354e4477055060eebfbd91e8793d86dce88d6410e630faf527

Comment (Wizdave97) [reply client/src/main.rs:109]: I'll set it to 150k gas, that seems to work.

**Review by seunlanlege — APPROVED** [review]

**Pull request merged**